pre-otp/.github/workflows/httpclient.yml

```diff
@@ -0,0 +1,39 @@
+name: HTTPCLIENT
```

Comment:
I worry about the pre-otp workflows becoming out of date since they won't be maintained. Should we delete pre-otp workflows?

Reply:
We certainly can, I only kept them so they'd be around with the old providers. I can just remove them